html/.storybook/preview-head.html

```diff
@@ -0,0 +1,25 @@
+<script>
+
+  const runOnPageChange = () => {
```

Comment:
might want to wrap the whole thing in an IIFE to avoid creating global vars

Reply:
@guern1kn tried this and found it doesn't work

Reply:
Gonna try a couple things first

Reply:
done!